scss/utilities/mixins/molecules/_footer-menus.scss

```diff
@@ -0,0 +1,37 @@
+@charset "UTF-8";
+
+//
+// Footer Menu Mixins
+//
+// Mixins for the menus that appear in the global footer.
+//
+// Style guide: Mixins.Molecules.Footer Menus
+//
+
+@mixin footer-menus {
+  margin: 0 0 1rem;
+}
+
+@mixin footer-menu {
```

Comment:
This would be best refactored down into a more common usage for a vertical menu.